wavelength-plugins/friends.js

```diff
@@ -0,0 +1,70 @@
+/****************************
+ * Friend List
+ * Original idea from WGC
+ * Coded by Volco
+ * *************************/
+'use strict';
+
+function onlineFriends(user) {
```

Comment:
I feel you should make this toggle-able.

Reply:
if this is toggle-able there's next to no point in the friends list system if you add friends, because then you don't know which of your friends is online when you log in

Reply:
Some people may like to add friends for their list but not want to be PMed every time they connect